iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/SasTokenProviderImpl.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tests.integration.com.microsoft.azure.sdk.iot.helpers;
+
+import com.microsoft.azure.sdk.iot.device.IotHubConnectionString;
+import com.microsoft.azure.sdk.iot.device.SasTokenProvider;
+import com.microsoft.azure.sdk.iot.device.auth.IotHubSasToken;
+
+import java.net.URISyntaxException;
+
+/**
+ * Basic implementation of the SasTokenProvider interface, for test purposes.
+ */
+public class SasTokenProviderImpl implements SasTokenProvider
```

Comment:
I assume this implementation works out of the box if they provide the connection string. Is that correct?

Reply:
It does, but this lives in our test code. I'd prefer users pick up the sample code's implementation rather than this one.